root/src/main/java/com/example/servingwebcontent/Movie.java

```diff
@@ -1,15 +1,19 @@
 package com.example.servingwebcontent;
 
 public class Movie {
-	private String id;
+	private String movieID;
```

Comment:
it doesn't need to be called "movieId" since "id" would suffice as it is implied that it is a movie id because it is in the Movie class.